tests/rptest/tests/raft_availability_test.py

```diff
@@ -222,7 +223,12 @@ def _expect_available(self):
         self._ping_pong()
         self.logger.info("Cluster is available as expected")
 
-    @cluster(num_nodes=3)
+    def setUp(self):
+        # Disable default RedpandaTest startup so that `redpanda_cluster`
+        # can change resource settings before starting cluster.
+        pass
+
+    @redpanda_cluster()
```

Comment:
this is cool. so by default i should think about redpanda_cluster as @cluster(num_nodes=3) and some other reasonable defaults? how does it interact with customizations done in RedpandaTest.__init__? I'm guessing the answer is that the annotations ends up taking precedence? Making setUp a noop is required when using @redpanda_cluster?

When should we reach for @cluster vs @redpanda_cluster?